app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointment.kt

```diff
@@ -34,9 +35,31 @@ data class OverdueAppointment(
     @Embedded(prefix = "phone_")
     val phoneNumber: PatientPhoneNumber?,
 
-    val isAtHighRisk: Boolean
+    /**
+     * Referencing a column alias in the same SQL query isn't allowed so
+     * mapping this index to [RiskLevel] is manually done using [riskLevel].
+     * Also see [isAtHighRisk].
+     */
+    val riskLevelIndex: Int
 ) {
 
+  @Ignore
+  val isAtHighRisk = riskLevelIndex <= RiskLevel.HIGH.levelIndex
+
+  @delegate:Ignore
+  val riskLevel by lazy {
```

Comment:
I added a comment explaining why this is lazy, but looks like I forgot to push it. Doing it now.

Reply:
Update: couldn't find my changes. The reason this is behind a lazy delegate is because `Enum#values()` allocates a new array everytime. Considering that `riskLevel` is only used in tests, I felt we shouldn't be allocating the array with every `OverdueAppointment` object.

Reply:
Why don't we just do a computed property instead of a lazy delegate then?

```
@Ignore 
val riskLevel: RiskLevel
  get() {
    RiskLevel.values().first { it.levelIndex == this.riskLevelIndex }
  }
```